ergo-mempool-sync/src/lib.rs

```diff
@@ -156,49 +167,68 @@ async fn sync<'a, TClient: ErgoNetwork>(client: &TClient, mut state: RefMut<'a,
     }
 }
 
-impl<'a, TClient, TChainSync> Stream for MempoolSync<'a, TClient, TChainSync>
-    where
-        TClient: ErgoNetwork,
-        TChainSync: Stream<Item=ChainUpgrade> + Unpin,
+pub enum Combined {
+    Mempool(MempoolUpdate),
+    Empty,
+}
```

Comment:
`Option<MempoolUpdate>`?

Reply:
I'm not sure, the reference is: https://github.com/spectrum-finance/spectrum/blob/1c03dcd9e643211c70990a683dffa22520b109ef/spectrum-network/tests/integration_tests/mod.rs#L328-L334